src/main/java/Duke/Helper/DeleteTask.java

```diff
@@ -0,0 +1,24 @@
+package Duke.Helper;
+
+import Duke.Duke;
+import Duke.Tasks.Task;
+
+import java.util.ArrayList;
+
+public class DeleteTask {
+    public static void deleteTask(ArrayList<Task> array, String message) {
```

Comment:
You might want to rename the attribute `array` since the name is too general and it is not clear what it represents.